### DISCUSSION TITLE
UpSnap behind NGINX Reverse Proxy with Basic Authentication

### DISCUSSION THREAD

**Discussion by rhasselbusch** [seriousm4x/UpSnap#261] [Help wanted]
Hi together,
i use UpSnap behind a NGINX Reverse Proxy with the URL wol.mydomain.com and it works great.
Because i want to use UpSnap internet facing, i'd like to enable Basic Authentication for this subdomain, but it doesn't seem to work with UpSnap or i'm missing some configuration.

What happens is that the Basic Authentication Popup appears and i enter the credentials. After hitting OK the dialog appears again and again, regardless how often i enter the correct credentials.
I tried enabling Websocket-Support for the reverse proxy subdomain, but nothing changed.

Is there a reasen (and maybe a soloution) why this constellation does not seem to work?

best regards
Rolf

**Comment by seriousm4x** [comment]
Hey, I've just tried it using nginx proxy manager and added basic auth. It works for me like expected, so I think there might be an error in your nginx config. Could you share it?

**Reply by rhasselbusch** [reply]
Yes of course, here is the .conf for the given host but i guess it is a Chrome related problem only, because i tested Safari and Firefox and those two browsers work as expected. Sorry for the formatting - the .conf-file format and the Code tag seems to conflict.

`# wol.myurl.cloud

server {
  set $forward_scheme http;
  set $server         "192.168.0.210";
  set $port           8090;

  listen 80;
listen [::]:80;

listen 443 ssl http2;
listen [::]:443 ssl http2;

  server_name wol.myurl.cloud;

  # Let's Encrypt SSL
  include conf.d/include/letsencrypt-acme-challenge.conf;
  include conf.d/include/ssl-ciphers.conf;
  ssl_certificate /etc/letsencrypt/live/npm-23/fullchain.pem;
  ssl_certificate_key /etc/letsencrypt/live/npm-23/privkey.pem;`

**Comment by rhasselbusch** [comment]
Okay, Firefox does not work also, it only has a bit different behaviour.
In Firefox the Auth Dialogue only appears once, but the site shows the error

Something went wrong while processing your request.

and shows none of the hosts.
So it seems to be my config which causes the error
